hasher-matcher-actioner/Makefile

```diff
@@ -24,6 +24,7 @@ upload_docker:
 
 terraform/terraform.tfvars:
 	echo 'hma_lambda_docker_uri = "${DOCKER_URI}:${DOCKER_TAG}"' > terraform/terraform.tfvars
+	echo 'organization = "${USER}-org"' >> terraform/terraform.tfvars
```

Comment:
not blocking: I don't like that this puts `user` in the name of things twice. Especially because that does not guarantees uniqueness really any better then `example-org` (Consider in the case of user='john'). 

Personally, if you don't want it to be required I would just give it a default value in terraform.tfvars.example

Reply:
The issue is that the user really needs to be aware about the need to set `organization`. As @BarrettOlson says, common names will break this. Even with `example-org` it's no different. I strongly prefer users have to set `organization` manually and that's called out in the docs so that it's clear as to why. (`organization` was introduced because some "scopes" of naming uniqueness in AWS are to the region, not to the AWS account.)

Blocking (until further discussion) don't default `organization`

Reply:
Removed default